src/usePopper.js

```diff
@@ -81,6 +85,7 @@ export default function usePopper(
     };
 
     popperInstanceRef.current = new PopperJS(referenceElement, popperElement, {
+      ...options,
```

Comment:
should this use an `options.popperConfig`? otherwise we'd need to spread out the properties under `options` that we consume above.

probably doing an `options.popperConfig` makes the typing easier, too

Reply:
options.placement, options.modifiers, options.eventsEnabled is separate props, options.popperConfig is composite property of the remaining options of Popperjs. I think this could confuse users.

Reply:
This is how e.g. `<Overlay>` works, though, no?

Reply:
shoot, my bad, that's _not_ how `<Overlay>` works. it _does_ spread in the popper config like you had here. sorry about that.

but if i'm not mistaken, then, i think in `usePopper`, `...popperConfig` should just be the options _not_ explicitly consumed?

Reply:
I didn't notice it.

Reply:
no, sorry, i meant your initial implementation was correct. my suggestion above was bad. we _should_ spread in `...popperOptions` into `usePopper`. we just need to also spread _out_ the options that we explicitly consume there. otherwise it would be a breaking change